pulpcore/pulpcore/app/serializers/task.py

```diff
@@ -115,10 +115,17 @@ class WorkerSerializer(ModelSerializer):
     )
 
     online = serializers.BooleanField(
-        help_text='Whether the worker is online or not. Defaults to True.',
+        help_text=_('Whether the worker is online'),
```

Comment:
Similar here, I'm hoping the values True and False will be explicitly called out with its case for any bool